packages/docs/docs/self-hosting/config-settings.md

```diff
@@ -106,34 +106,35 @@ For example, if your environment name is "prod", then the "baseUrl" parameter na
+| `port`                 | The port number that the API server binds to inside the Docker image. By default, you should use `8103`. In some cases, you may need to use `5000`.                                                   | yes      | `init`     | `8103`                                               |
+| `baseUrl`              | The fully qualified base URL of the API server including a trailing slash. For example, `https://api.example.com/`.                                                                                   | yes      | `init`     |                                                      |
+| `issuer`               | The JWK issuer. By default, Medplum server uses built in OAuth, so `issuer` should be the same as `baseUrl`.                                                                                          |          |            | `baseUrl`                                            |
+| `jwksUrl`              | The JWKS URL. By default, Medplum server uses built in OAuth, so `jwksUrl` should be `baseUrl` + `.well-known/jwks.json`.                                                                             |          |            | `baseUrl` + `.well-known/jwks.json`                  |
+| `authorizeUrl`         | The OAuth authorize URL. By default, Medplum server uses built in OAuth, so `authorizeUrl` should be `baseUrl` + `oauth2/authorize`.                                                                  |          |            | `baseUrl` + `oauth2/authorize`                       |
+| `tokenUrl`             | The OAuth token URL. By default, Medplum server uses built in OAuth, so `tokenUrl` should be `baseUrl` + `oauth2/token`.                                                                              |          |            | `baseUrl` + `oauth2/token`                           |
+| `userInfoUrl`          | The OAuth userinfo URL. By default, Medplum server uses built in OAuth, so `userInfoUrl` should be `baseUrl` + `oauth2/userinfo`.                                                                     |          |            | `baseUrl` + `oauth2/userinfo`                        |
+| `appBaseUrl`           | The fully qualified URL of the user-facing app. This is used for CORS and system generated emails. For example, `https://app.example.com/`.                                                           | yes      | `init`     |                                                      |
+| `logLevel`             | Verbosity of logging: `'NONE'`, `'ERROR'`, `'WARN'`, `'INFO'`, `'DEBUG'`                                                                                                                              |          |            | `'ERROR'` if `NODE_ENV == 'test`. `'INFO'` otherwise |
```

Comment:
For default value, I think we can just say `INFO` -- `NODE_ENV == 'test'` is only when running unit tests.